app/views/art_items/_form.html.erb

```diff
@@ -66,7 +66,7 @@
   </div>
 
   <div id="attached_files" class="my-5">
-    <%= form.label :documents, "Documents (pdf,txt,jpg,png,doc)" %><span class="optional-text"> * optional</span>
+    <%= form.label :documents %><span class="optional-text">(5 Mb limit per item) * optional</span>
```

Comment:
I actually liked the way you added the accepted file types earlier. How about adding them back next to the documents label ? If it just says documents, a user will upload any kind of document and then will figure out that it is not allowed. I think it is better to have all the information available to the user.

Reply:
did it